alembic/ddl/impl.py

```diff
@@ -39,6 +44,7 @@ class DefaultImpl(with_metaclass(ImplMeta)):
 
     transactional_ddl = False
     command_terminator = ";"
+    type_synonyms = [{"NUMERIC", "DECIMAL"}]
```

Comment:
**mike bayer** (zzzeek) wrote:

I tend to use tuples for things like this to make sure nothing is mutated...